apps/settings/locales/settings.en-US.properties

```diff
@@ -1130,8 +1130,9 @@ draw-tile-borders=Draw tile borders
 draw-layer-borders=Draw layer borders
 log-animations=Log slow animations
 remote-debugging-usb=Debugging via USB
-dev-tools-wifi=DevTools via Wi-Fi
-dev-tools-qps=Pseudo-localization
+devtools-wifi=DevTools via Wi-Fi
```

Comment:
Please keep in mind that changing `l10n-id`s causes re-translation work, which is unnecessary when there is no string change.